lib/SyTest/Homeserver.pm

```diff
@@ -566,13 +586,11 @@ sub _run_command
             $failure = "$cmd failed $exitcode";
          }
 
-         if( $stderr ) {
-            $failure .= ": $stderr";
-         }
          $fut->fail( $failure );
-      }
+      },
    );
 
+   $self->loop->add($proc);
```

Comment:
I'm not sure what the purpose of this line is?

Reply:
heh, nor am I really. It's just some incantation you have to do when you use `IO::Async::Process`. See the examples at https://metacpan.org/pod/IO::Async::Process#SYNOPSIS.